src/targets/gpu/convolution.cpp

```diff
@@ -145,9 +144,11 @@ shape miopen_convolution::find(context& ctx, const shape& output_shape, std::vec
     if(status != miopenStatusSuccess)
         MIGRAPHX_THROW("MIOpen Convolution: get solution failed");
 
-    solution_id = solutions.front().solution_id;
+    solution_id    = solutions.front().solution_id;
```

Comment:
So you could assign `solutions.front()` to a local variable. Actually, it could probably be refactored to only get 1 solution instead of a vector.

Reply:
How can i make it return single `solution` ?

Reply:
Done, Removed call to `miopenGetSolutionCount`.